spec/models/keyword_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Keyword, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
We can add `name` attribute validation test like https://github.com/nimblehq/the1-web/blob/f97fcb48e50e605a92b35398dbd8e3ec6324a990/spec/models/the1_contact/partner_spec.rb#L7

Reply:
Nice example :) I've added the expect presence of name for keyword too :) 
Thanks!